mantle/kola/harness.go

```diff
@@ -1340,6 +1349,7 @@ func runTest(h *harness.H, t *register.Test, pltfrm string, flight platform.Flig
 	h.SetSubtests(t.Subtests)
 
 	rconf := &platform.RuntimeConfig{
+		AllowFailedUnits:   testSkipBaseChecks(t),
```

Comment:
Would be nice to update the docs in `external-tests.md`.

Reply:
I'll just update the PR and merge (no need for another CI run since I'm just updating documentation).

Reply:
done